tests/unit/test_r.py

```diff
@@ -7,20 +7,6 @@
 from repo2docker import buildpacks
 
 
-def test_unsupported_version(tmpdir):
```

Comment:
Does the removal of this test mean all versions are supported?

Reply:
YESSS! basically anything that has a deb produced by RStudio is now supported